R/get_mangrove.R

```diff
@@ -79,7 +80,6 @@ NULL
       exdir = rundir,
     )
     shp <- file.path(rundir, "GMW_2007_v2.0.shp")
-    write_sf(shp, file.path(rundir, basename("gmw-extent_2007.gpkg")))
```

Comment:
I just caught this change. We better have no gdal system call in the package. That caused me a lot of headaches for the automatic checks on windows. Could you please go back to use sf for this? And if that applies for other functions, there as well.

Reply:
Okay, I think this one is only for the mangrove function. 

And one thing, the tests seem okay here but whenever I test, the tests of get functions are getting failed. So, I will push only the files which I changed, is it okay?

Reply:
So the current state of the get tests fail on your machine? Because they  run successfully on github actions. What errors do you get?

Reply:
1. test portfolio - "portfolio_out.gpkg" about this tmpfile
2. old vs new snapshots for all other remaining get functions

Reply:
Ok, about the snapshots, have you pulled latest changes to the indicator branch?

Reply:
yes, I started fresh project from indicator branch and still failing the devtools test

Reply:
Ok, no worries. We'll figure it out. I guess you can start pushing new indicator functions + their tests?